### DISCUSSION TITLE
[Improvement] Replace Prettier and Eslint with Biome and add pre-commit hooks

### DISCUSSION THREAD

**Discussion by santigandolfo** [fastapi/full-stack-fastapi-template#1107] [Questions] [answered]
In my case I forked the project and replaced Prettier and Eslint with Biome. This change also allowed me to run linting and formating in the frontend pre-commit hooks with virtually no impact when trying to commit changes.

If I made a PR adding those changes, would you consider merging it or would prefer keeping Prettier and Eslint? What about just adding pre-commit hooks for those? My only problem with that is that they are considerably slower and I hate it when trying to make a commit takes too long.

**Comment by alejsdev** [comment, marked as answer]
Hi @santigandolfo, we appreciate your initiative in optimizing the project. Right now we're not rigidly tied to Prettier and ESLint and we're open to exploring alternatives like Biome. We would review your PR with the proposed changes. Thanks for your interest 😄
